src/app/js/formats/injectData.js

```diff
@@ -108,11 +109,17 @@ export default url => FormatView => {
                 field,
                 isLoaded,
                 error,
+                resource,
                 ...props
             } = this.props;
 
             if (error) {
-                return (
+                return error === 'bad value' ? (
+                    <InvalidFormat
+                        format={field.format}
+                        value={resource[field.name]}
+                    />
+                ) : (
                     <p style={styles.message}>{polyglot.t('chart_error')}</p>
```

Comment:
Please use `AdminOnlyAlert` component to hide this error message when logged as a user

Reply:
No, I think the user would want to know he has connexion problem